Library/Formula/pygtk.rb

```diff
@@ -1,9 +1,8 @@
-require 'formula'
-
 class Pygtk < Formula
-  homepage 'http://www.pygtk.org/'
-  url 'http://ftp.gnome.org/pub/GNOME/sources/pygtk/2.24/pygtk-2.24.0.tar.bz2'
-  sha1 '344e6a32a5e8c7e0aaeb807e0636a163095231c2'
+  homepage "http://www.pygtk.org/"
+  url "http://ftp.gnome.org/pub/GNOME/sources/pygtk/2.24/pygtk-2.24.0.tar.bz2"
```

Comment:
And this one too:
https://download.gnome.org/sources/pygtk/2.24/pygtk-2.24.0.tar.bz2